tests/src/test/java/org/jboss/windup/tests/bootstrap/AbstractBootstrapTestWithRules.java

```diff
@@ -27,7 +27,7 @@
             + "        <targetTechnology id=\"eap7\" versionRange=\"[7,)\" />\n"
             + "        <tag>test-tag-eap</tag>\n"
             + "    </metadata>\n"
-            + "\n"
+            + System.lineSeparator()
             + "    <rules>\n"
```

Comment:
I wonder why are you changing one \n while leaving the others?

Reply:
no big reason, just not catch them, but this is test rules, so no big deal